sdk/documentintelligence/azure-ai-documentintelligence/setup.py

```diff
@@ -38,7 +38,7 @@
     url="https://github.com/Azure/azure-sdk-for-python/tree/main/sdk",
     keywords="azure, azure sdk",
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```

Comment:
might as well add 3.13 support as well

Reply:
Supported python versions also come from the generator so if someone tries to regen again over what we check in, python 3.13 would be removed until it's officially added by the generator as well. So I'd rather we wait for the generator to output it to not cause inconsistencies in the future. I'll open an issue for them to add it.